browser/ui/toolbar/brave_app_menu_model_browsertest.cc

```diff
@@ -85,6 +93,16 @@ IN_PROC_BROWSER_TEST_F(BraveAppMenuBrowserTest, MenuOrderTest) {
   std::vector<int> commands_disabled_for_normal_profile = {
     IDC_NEW_TOR_CONNECTION_FOR_SITE,
   };
+#if BUILDFLAG(ENABLE_BRAVE_SYNC)
+  if (!switches::IsSyncAllowedByFlag()) {
```

Comment:
nit: How about making this logic as a method because same codes are duplicated many times.

Reply:
@simonhong probably makes sense, but I'm just copy pasting code that was removed here. Improvements can be done as a follow-up.